src/Symfony/Component/Console/Application.php

```diff
@@ -1080,6 +1088,12 @@ private function stringWidth($string)
         return mb_strwidth($string, $encoding);
     }
 
+    /**
+     * @param string $string
+     * @param int    $width
+     *
+     * @return string[]
```

Comment:
kind of the same, the added value for the types helps with SCA and with that IDE's, but sure, happy to ditch